isis/src/base/apps/isis2pds/isis2pds.cpp

```diff
@@ -0,0 +1,246 @@
+#include <QString>
+
+#include "Cube.h"
+#include "ExportDescription.h"
+#include "FileName.h"
+#include "isis2pds.h"
+#include "Process.h"
+#include "ProcessExportPds.h"
+#include "ProcessExportPds4.h"
+#include "Pvl.h"
+#include "PvlKeyword.h"
+#include "PvlToXmlTranslationManager.h"
+
+
+using namespace std;
+namespace Isis{
+
+  enum Pixtype { NONE, NEG, BOTH };
+
+  void setRangeAndPixels(UserInterface &ui, ProcessExport &p,
+                         double &min, double &max, Pixtype ptype);
+
+  void isis2pds(UserInterface &ui, Pvl *log){
+    Process p;
+
+    CubeAttributeInput cai;
+    Cube *icube = p.SetInputCube(ui.GetFileName("FROM"), cai, ReadWrite);
```

Comment:
You should copy over the `setVirtualBands()` call from another converted app. This makes it so that the newly created cube `icube` will have the correct number of bands when the user runs the program from the command line. We also don't need to call `SetInputCube` and `endProcess` here since  they are called in the second isis2pds function.